packages/lib/src/components/components-name-map.ts

```diff
@@ -0,0 +1,161 @@
+import { TxVariants } from './tx-variants';
+
+/**
+ * Maps each payment type with the string name of its Component element.
+ *
+ * It is used to give nicer error messages when creating components using Drop-in.
+ * Example: paymentMethods response contains blik, but Blik component is not configured. So it warns that 'Blik' must be imported and added to the Drop-in configuration.
+ */
+const componentsNameMap = {
```

Comment:
Should this start with a capital letter: `ComponentsNameMap` (to keep it in line with how you've named `ComponentsMap`?